src/components/ui/ExternalButtonLink.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <a class="button button_link" :href="href">
```

Comment:
I don't know if 'button_link' was copy pasted or something, but that schema doesn't fit in BEM. Revisit?